save-demo/src/main/kotlin/com/saveourtool/save/demo/utils/KubernetesUtils.kt

```diff
@@ -128,17 +144,26 @@ fun jobNameForDemo(demo: Demo) = with(demo) { "demo-${organizationName.lowercase
 private fun demoAgentContainerSpec(
     imageName: String,
     agentDownloadUrl: String,
+    demo: Demo,
     kubernetesSettings: KubernetesConfig,
+    agentConfig: ConfigProperties.AgentConfig,
 ) = Container().apply {
     name = "save-demo-agent-pod"
     image = imageName
     imagePullPolicy = "IfNotPresent"
 
     val envOptions = sequenceOf(
         "KTOR_LOG_LEVEL" to "TRACE",
+        DemoAgentConfig.DEMO_URL_ENV to agentConfig.demoUrl,
```

Comment:
can we provide it as a single url to save-demo-agent instead of constructing an url on agent side?

Reply:
Ok, good idea, will do it

Reply:
Done

Reply:
Renamed the variable and started to pass the whole url instead of just `baseUrl`